cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailureDetector.java

```diff
@@ -177,12 +181,22 @@ private void parsePersistedFailedBrokers(String failedBrokerListString) {
     }
   }
 
+  private boolean tooManyFailedBrokers(int deadBrokerCount, int aliveBrokerCount) {
+    return deadBrokerCount > FIXABLE_BROKER_FAILURE_UPPER_BOUNDARY ||
+           (double) deadBrokerCount / (deadBrokerCount + aliveBrokerCount) > FIXABLE_BROKER_FAILURE_PERCENTAGE_UPPER_BOUNDARY;
+  }
+
   private void reportBrokerFailures() {
     if (!_failedBrokers.isEmpty()) {
-      Map<String, Object> parameterConfigOverrides = new HashMap<>(3);
+      Map<String, Object> parameterConfigOverrides = new HashMap<>(4);
       parameterConfigOverrides.put(KAFKA_CRUISE_CONTROL_OBJECT_CONFIG, _kafkaCruiseControl);
-      parameterConfigOverrides.put(FAILED_BROKERS_OBJECT_CONFIG, failedBrokers());
+      Map<Integer, Long> failedBrokers = failedBrokers();
+      parameterConfigOverrides.put(FAILED_BROKERS_OBJECT_CONFIG, failedBrokers);
       parameterConfigOverrides.put(ANOMALY_DETECTION_TIME_MS_OBJECT_CONFIG, _kafkaCruiseControl.timeMs());
+      // IF there are too many broker fails at the same time, mark the anomaly as unfixable.
```

Comment:
This comment seems inconsistent with the operation -- i.e. this piece of code just sets a config value. 
Can we move this comment to the relevant function that interprets this config?